### PR TITLE
update TODO

### PR DESCRIPTION
**Pull request by evverx**

### CODE REVIEW

**Comment by keszybz** [comment]
Meh, do we really want to put this in TODO? If though github issues are the new TODO list :)

File: TODO [hunk 1]
```diff
@@ -727,6 +727,9 @@ Features:
     but much rather a disconnect on success.
   - when breaking cycles drop sysv services first, then services from /run, then from /etc, then from /usr
   - when a bus name of a service disappears from the bus make sure to queue further activation requests
+  - use *_raw API from libselinux
+    https://github.com/systemd/systemd/pull/1107
+    https://github.com/systemd/systemd/pull/2797
```
Comment (zonque) [inline TODO:732]: I see no point in adding PR URLs to `TODO` really. And about the topic in general, I don't know what's left in this area. @keszybz ?

**Comment by evverx** [comment]
OK:) So, closing this now.

File: TODO (hunk 1 above)
Comment (evverx) [inline TODO:732]: > I don't know what's left in this area

https://github.com/systemd/systemd/pull/2797#issuecomment-197329718

**Comment by poettering** [comment]
Yeah, PRs and issues are enough for a TODO. If something has an open PR or issue then that's enough to keep track of it.

**Pull request closed without merging**